main.lua

```diff
@@ -14,23 +14,98 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
---]]
-
-local bundle = require('luvi').bundle
+--]] local bundle = require("luvi").bundle
```

Comment:
What happened here? This should stay on its own line.

Reply:
Autoformat (LuaFormatter) seems to be messed up. Haven't found an option to configure this, yet. I'll try again.